### ISSUE TITLE
Cordova File utilities cannot access steroids.app.absolutePath on Standalone builds

### ISSUE THREAD

**Issue by pre** [AppGyver/steroids#817]
Cordova File plugin and related utilities cannot access `steroids.app.absolutePath` on Standalone builds. Files are accessible on Scanner, though.

As a workaround:

Store your sound file (or SQLite database or such) in `steroids.app.absoluteUserFilesPath`. 

There's a hidden feature in Android that will allow you to copy files from your APK file to the device's storage on application start. Copying is done only when the file with the same name does NOT exist in the user files. 

Place `steroids.config.copyToUserFiles = ["test.txt", "test.png"]` in your application.coffee and those files will be available from `steroids.app.absoluteUserFilesPath` (and from `http://localhost`). Paths are relative to the `dist/` folder of your Steroids application.

**Comment by fidoboy** [comment]
Thanks for the tip @pre. I've a question, if I add that line into my application.coffee these files are also being copied from the iOS version, and I want to avoid that, cause I haven't this problem with iOS. There is any way to perform this action for Android version only?

**Comment by pre** [comment]
Have you experienced the behaviour that these files are being copied in iOS?

This hidden feature is currently only implemented in Android.

**Comment by fidoboy** [comment]
Not really, I was only asking. If it doesn't work on iOS it's fine for me. Thanks

**Comment by harsaharsa** [comment]
Marking this as `wontfix` since it's so deeply rooted in how Cordova File plugin works. A fix would include our own File API.

**Comment by fidoboy** [comment]
Thanks @harsaharsa, no problem for me cause it's not very critical; as @pre commented, it can be solved using a simple workaround. It's not the most desirable way, but it works :+1:
